<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_p</class>
 <widget class="QDialog" name="add_p">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create new period</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>379</width>
     <height>539</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_p_4" stretch="0,5,0">
    <property name="spacing">
     <number>6</number>
    </property>
    <item>
     <widget class="QLabel" name="objects2_4">
      <property name="text">
       <string>All periods:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget2_4"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_12" stretch="0,0">
      <item>
       <widget class="QLabel" name="lselected_5">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="3,9">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item>
           <widget class="QLabel" name="lplname_4">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lpltime_4">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_7">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>58</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item>
           <widget class="QLabel" name="lpname_4">
            <property name="font">
             <font>
              <family>Ubuntu</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lptime_4">
            <property name="font">
             <font>
              <family>Ubuntu</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_8">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>58</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>600</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>600</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
